fk/secretsend_test.go

```diff
@@ -79,3 +88,266 @@ func assertMessageBodyMatchesSecretContent(unverifiedBody io.Reader, secret stri
 		t.Fatalf("recovered message incorrect got '%s', want '%s'", messageBuf.Bytes(), secret)
 	}
 }
+
+const (
+	englishAlphabet     = "AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz"
+	printableCharacters = "☠☮☯♠Ω♤♣♧♥♡♦♢♔♕♚♛⚜★☆✮✯☄☾☽☼☀☁☂☃☻☺☹۞۩ εїзƸ̵̡Ӝ̵̨̄ƷξЖЗεжз☎☏¢☚☛"
+	sampleEmojis        = "😀😁😂🤣😃😄😅😆😉😊😋😎😍😘😗😙😚🙂"
+	breaks              = "line 1\rline 2\n"
+)
+
+func TestContainsDisallowedRunes(t *testing.T) {
```

Comment:
Because the function signature is not a valid test function, these tests don't actually run

Reply:
errr

Reply:
I'm talking bollocks, ignore me.

Reply:
🤣